docs/examples/Configuring_QCoDeS.ipynb

```diff
@@ -359,8 +360,8 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "one can change the core values at runtime, but there is no guarantee that they are valid.\n",
-    "Changing core values will also shadow the config from being updated if the defaults are updated upsteam (i.e. in the repo). So use it with a care."
+    "One can change the core values at runtime, but there is no guarantee that they are valid.\n",
+    "Changing core values will also shadow the configuration from being updated if the defaults are updated in the upsteam (i.e. in the repository). So use it with a care."
```

Comment:
this sentences could be made more clear. for example, smth like "Since user config shadows the default config that comes with QCoDeS, apply care when changing values under `core` section. That section is meant for settings that are determined by QCoDeS core developers. .....bla .... bla ...bla"